src/bin/casr-cluster.rs

```diff
@@ -349,6 +359,19 @@ pub fn make_clusters(inpath: &Path, outpath: Option<&Path>, jobs: usize) -> Resu
     Ok(cluster_cnt)
 }
 
+fn dedup_stacktraces(stacktraces: &[Stacktrace]) -> Vec<Option<Stacktrace>> {
```

Comment:
Docs + separate module: `deduplication`  as funciton name `dedup'?

Reply:
moved to stacktrace.rs